manifests/init.pp

```diff
@@ -105,11 +106,18 @@
     require => Package['eyaml'],
   }
 
-  exec { 'eyaml_createkeys':
-    path    => $createkeys_path,
-    command => "eyaml createkeys --pkcs7-private-key=${private_key_path} --pkcs7-public-key=${public_key_path}",
-    creates => $private_key_path,
-    require => File['eyaml_keys_dir'],
+  if $manage_keys {
```

Comment:
This is not required since `if $manage_keys {`does this validation if we ensure it is a bool.